plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/UIUtils.java

```diff
@@ -577,6 +577,25 @@ public static Font modifyFont(@NotNull Font normalFont, int style) {
         return new Font(normalFont.getDevice(), data);
     }
 
+
+    /**
+     * Modifies the size of the given font by applying the specified modifier to the current font size.
+     *
+     * @param normalFont the original font whose size needs to be modified. Must not be null.
```

Comment:
It's already clear from the contract provided by annotations
```suggestion
     * @param normalFont the original font whose size needs to be modified.
```